Types/Entities/World.js

```diff
@@ -293,9 +295,24 @@ function World(spec) {
     requestDraw()
   }
 
+  function formatTime(t) {
+    // t is a decimal number of seconds, and we want to format it as ss.ms
+    const ms = Math.floor((t % 1) * 100)
+    const ss = Math.floor(t)
+    return ss == 1 ? '1 second' : `${ss}.${ms} seconds`
+  }
+
   function setCompletionTime(t) {
     completionTime = t
-    ui.completionTime.innerHTML = t
```

Comment:
This is an issue here - this is actually the interface that the scoring service is using to retrieve the exact time of runs using Puppeteer - it was hidden because it is not used for UI purposes.  This is a breaking change as a result.  Please add a new separate completionTime div element for the user interface you're adding here.

Reply:
Ah, gotcha. will fix 👍

Reply:
Can we not just use the div in the victory screen for this purpose @grymmy?

Reply:
> Can we not just use the div in the victory screen for this purpose @grymmy?
This breaks existing functionality and is a simple change.  The burden is on the PR author to not break existing stuff, and this should be a very simple change.